library/src/main/java/com/dslplatform/json/DslJson.java

```diff
@@ -816,6 +821,8 @@ public final Object getDefault(@Nullable Type manifest) {
 
 	private final ConcurrentMap<Type, JsonReader.ReadObject<?>> readers = new ConcurrentHashMap<Type, JsonReader.ReadObject<?>>();
 	private final ConcurrentMap<Type, JsonReader.BindObject<?>> binders = new ConcurrentHashMap<Type, JsonReader.BindObject<?>>();
+    private final ConcurrentMap<Type, JsonWriter.WriteObject<?>> writers = new ConcurrentHashMap<Type, JsonWriter.WriteObject<?>>();
```

Comment:
There are some writers variables around. We should rename them (the other ones)

Reply:
I found few local variables with same name. I've renamed them